test/e2e/util/util.go

```diff
@@ -195,6 +201,100 @@ func WaitForGuestKubeConfig(t *testing.T, ctx context.Context, client crclient.C
 	return data, nil
 }
 
+func WaitForKubeVirtMachines(t *testing.T, ctx context.Context, client crclient.Client, hostedCluster *hyperv1.HostedCluster, count int32) {
```

Comment:
can you please move this into a kubevirt file?

Reply:
done